data/transit/route/bus.json

```diff
@@ -636,6 +636,7 @@
         "network": "APT",
         "network:wikidata": "Q3601116",
         "network:wikipedia": "it:APT Gorizia",
+        "operator": "TPL FVG"
```

Comment:
Just an FYI, there is a missing comma at the end of this line, causing a build error.

Reply:
see https://github.com/osmlab/name-suggestion-index/pull/5716/commits/480f13dac5543b57afd4839f4b17261ec141bfdb